### ISSUE TITLE
Completion doesn't work from test files

### ISSUE THREAD

**Issue by manuranga** [ballerina-platform/ballerina-lang#17870]
```
Version: 1.37.1 (user setup)
Commit: f06011ac164ae4dc8e753a3fe7f9549844d15e35
Date: 2019-08-15T16:17:55.855Z
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.17134
```
![Code_autmHXKN3c](https://user-images.githubusercontent.com/1686124/63159226-9d397200-c038-11e9-9b6f-7d848aee60ef.png)

**Comment by nadeeshaan** [comment]
This is due to the current testerina issues, and will be solved once the testerina work is finalized

**Comment by nadeeshaan** [comment]
This has been fixed and shipped with v1.0.0, hence closing the issue
